semgrep/tests/e2e/conftest.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+from pathlib import Path
+from typing import Optional
+from typing import Sequence
+from typing import Union
+
+import pytest
+
+
+E2E_TESTS_PATH = Path(__file__).parent
+
+
+def _run_semgrep(
+    config: Optional[Union[str, Path]] = None,
+    *,
+    target_name: str = "basic",
+    stderr: bool = False
+) -> str:
+    config_opts = ["--config", config] if config is not None else []
+
+    return subprocess.check_output(
```

Comment:
This is calling Python code from Python code. Is there any way we can just call this library directly instead of `subprocess`ing out?

Reply:
I think it fits with the end-to-end philosophy to test whether commands run as expected. In `tests/unit/` we definitely shouldn't have a single usage of `subprocess` though. Does that sound okay?